lib/opencensus_phoenix/instrumenter.ex

```diff
@@ -58,12 +62,57 @@ defmodule OpencensusPhoenix.Instrumenter do
     :ocp.with_span_ctx(parent_span_ctx)
   end
 
-  def span_name(conn) do
+  def span_name(conn, :controller_action, _) do
```

Comment:
Why is that public?

Reply:
I suspect that it shouldn’t be, I just didn’t notice that it was when I made my changes :)

Reply:
Made it private.